packages/allow-deny/allow-deny.js

```diff
@@ -468,15 +454,15 @@ function docToValidate(validator, doc, generatedId) {
 
 function addValidator(collection, allowOrDeny, options) {
   // validate keys
-  const validKeysRegEx = /^(?:insert|update|remove|fetch|transform)$/;
+  const validKeysRegEx = /^(?:insertAsync|updateAsync|removeAsync|fetch|transform)$/;
   Object.keys(options).forEach((key) => {
     if (!validKeysRegEx.test(key))
       throw new Error(allowOrDeny + ": Invalid key: " + key);
   });
 
   collection._restricted = true;
 
-  ['insert', 'update', 'remove'].forEach((name) => {
+  ['insertAsync', 'updateAsync', 'removeAsync'].forEach((name) => {
```

Comment:
Is this breaking change necessary? It seems this is only changing the property names and it still works the same. If we want to make this change, maybe it could still support the old property names too as an alias for the new names.

Reply:
I [added to it](https://github.com/meteor/meteor/blob/e5e1ad2349272b8dead4b84e3f82bd99a9747609/packages/allow-deny/allow-deny.js#L680-L687) instead of removing the older ones. We need all of them.